generic/tests/cfg/vlan.cfg

```diff
@@ -18,7 +17,12 @@
     RHEL.5:
         cmd_type = vconfig
         msg_pattern ='ERROR: trying to add VLAN #4095 to IF -:\w+:-  error: Invalid argument'
+    Windows:
+        vlan_num = 1
+        win_vlan_id = 900
+        set_vlan_cmd = powershell -command "Set-NetAdapter -Name '%s' -VlanID ${win_vlan_id} -Confirm:$false"
     variants:
         - @vlan_connective_test:
```

Comment:
is this also only for linux?

Reply:
Thanks for review. 
No, I want to do ping test for both windows and linux, windows guest can do ping test on this variant.